pages/download.vue

```diff
@@ -33,6 +33,11 @@
                     <h1>🎉 Congratulations!</h1>
                     You have successfully created an <a href="https://neurobagel.org/dictionaries/" target="_blank">neurobagel annotated .json data dictionary</a>.
                 </b-alert>
+                <h3>Data Dictionary</h3>
+                <p>
+                Here is the final .json data dictionary that you have created:
+                </p>
```

Comment:
Right now, this is always visible. That'd be confusing if the data dictionary is not valid and therefore nothing is shown in the next block.

Reply:
The code now handles valid and invalid data dictionaries properly. If the data dictionary is invalid, it notifies the user and offers appropriate information.

Reply:
awesome, thank you very much @Manya-tech! Do you feel comfortable to add a test for this new functionality you added? The test should probably live here: https://github.com/neurobagel/annotation_tool/blob/main/cypress/e2e/page/download-pagetests.cy.js

If no, we can also add the test for you by taking over the PR.